CHANGELOG-developer.asciidoc

```diff
@@ -33,6 +33,7 @@ The list below covers the major changes between 6.3.0 and master only.
 ==== Bugfixes
 
 - Fix permissions of generated Filebeat filesets. {pull}7140[7140]
+- Fix issue on asset generation that could lead to different results in Windows. {pull}8464[8464]
```

Comment:
Is `CHANGELOG-developer.asciidoc` the correct doc to target? 

I notice other PRs like https://github.com/elastic/beats/pull/8147/files target `CHANGELOG.asciidoc`

Reply:
It is a change only for development, it doesn't affect any functionality.